### ISSUE TITLE
Components inventory

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Angular#169]
## Missing from ng-bootstrap

Some of them matches Boosted custom components or feature request, so it should probably be included. However we have to double-check accessibility.

- [DatePicker](https://ng-bootstrap.github.io/#/components/datepicker/overview)
- [Rating](https://ng-bootstrap.github.io/#/components/rating/examples)
- [Table](https://ng-bootstrap.github.io/#/components/table/overview)
- [TimePicker](https://ng-bootstrap.github.io/#/components/timepicker/examples)
- [Typeahead](https://ng-bootstrap.github.io/#/components/typeahead/examples)

## Equivalents

Those two are custom in ng-boosted, and ignored from ng-bootstrap: couldn't we merge them?

- [Button](https://ng-bootstrap.github.io/#/components/buttons/examples) look like [Toggle](https://ng-boosted.orange.com/#/documentation/o-toggle)
- [Carousel](https://ng-bootstrap.github.io/#/components/carousel/examples) is replaced with [Swiper](https://ng-boosted.orange.com/#/documentation/swiper)

## Only in ng-boosted

Nothing much to say, however we should ensure their accessibility.

- [Calendar](https://ng-boosted.orange.com/#/documentation/calendar)
- [Scroll Top](https://ng-boosted.orange.com/#/documentation/o-scroll-top)
- [Switch](https://ng-boosted.orange.com/#/documentation/o-switch)

**Comment by BehnooshShiva** [comment]
**here are the components that have been removed :**
Calendar : lib-o-calendar, Navbar : lib-o-navbar, o-nav-menu & o-nav-link, Scroll Top : llib-scroll-top, Swiper : lib-o-carousel, Switch : lib-o-switch, Toast : lib-o-toast, Toggle

**And here are the equivalent components :**
Datepicker, navbar, Dropdown, Scroll Up, Carousel , Toast
Please find these components from version 5.2.0
